ebos/eclproblem.hh

```diff
@@ -688,7 +688,10 @@ public:
         }
         }
         bool isSubStep = !EWOMS_GET_PARAM(TypeTag, bool, EnableWriteAllSolutions) && !this->simulator().episodeWillBeOver();
-        eclWriter_->evalSummaryState(isSubStep);
+        // For CpGrid with LGRs, ecl/vtk output is not supported yet.
+        if (enableEclOutput_) {
+            eclWriter_->evalSummaryState(isSubStep);
+        }
```

Comment:
When I recommended  doing the PR, I forgot about this one. Sorry
This will omit the printing of e.g. injection and production reports when using `--enable-ecl-output=false`. Somebody else need to comment here whether this part can be merged. @bska @atgeirr?